bootstrap.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
+# The -e option would make our script exit with an error if any command
+# fails while the -x option makes verbosely it output what it does
+
+# Install Pipenv, the -n option makes sudo fail instead of asking for a
+# password if we don’t have sufficient privileges to run it
+sudo -n dnf install -y pipenv
+
+cd /vagrant
+# Install dependencies with Pipenv
+pipenv sync
+
+# Run database migrations
+pipenv run python manage.py migrate
+
```

Comment:
Im not sure that this is necessary, 
Please check if you need it...

Reply:
the entire file or part of it? @bardielle

Reply:
only line 14

Reply:
changed it @bardielle

Reply:
:+1:

Reply:
done :)